alerts/couchbase/high-memory-usage.json

```diff
@@ -0,0 +1,36 @@
+{
+    "displayName": "Couchbase - High Bucket Memory Usage",
+    "documentation": {
+        "content": "If `bucket.memory.usage` bytes is higher than what is anticipated it could show that the bucket needs to be allocated more. Subjective to environment and this alert can be customized for your environment.",
+        "mimeType": "text/markdown"
+    },
+    "userLabels": {},
+    "conditions": [
+        {
+            "displayName": "VM Instance - workload/couchbase.bucket.item.ejection.count",
+            "conditionThreshold": {
+                "filter": "resource.type = \"gce_instance\" AND metric.type = \"workload.googleapis.com/couchbase.bucket.memory.usage\"",
+                "aggregations": [
+                    {
+                        "alignmentPeriod": "300s",
+                        "crossSeriesReducer": "REDUCE_NONE",
+                        "perSeriesAligner": "ALIGN_MEAN"
+                    }
+                ],
+                "comparison": "COMPARISON_GT",
+                "duration": "30s",
```

Comment:
condition_threshold.duration had an invalid value of \"30s\": only multiples of 1 minute are supported